### PR TITLE
Add cph + Title Case

### PR DESCRIPTION
**Pull request by maelle**
cf https://github.com/DavisVaughan/extrachecks

### CODE REVIEW

**Comment by krlmlr** [comment]
Thanks! Would you like to add yourself as a contributor?

**Comment by maelle** [comment]
Thanks, done!

File: DESCRIPTION [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
-             comment = c(ORCID = "0000-0003-0748-6624")))
-Description: Streamlines the process of updating changelogs
-    (NEWS.md)
+             comment = c(ORCID = "0000-0003-0748-6624")),
+      person(given = "Maëlle", 
+             family = "Salmon",
+             role = "ctb",
+             email = "msmaellesalmon@gmail.com",
+             comment = c(ORCID = "0000-0002-2815-0399")))
+Description: Streamlines the process of updating 'changelogs'
```
Comment (krlmlr) [inline DESCRIPTION:21]: Can you please use a technique similar to RPostgres to get rid of the 'quotes'?

https://github.com/r-dbi/RPostgres/tree/main/.aspell

Comment (krlmlr) [reply DESCRIPTION:21]: Thanks to @jimhester!

**Comment by maelle** [comment]
merging for doing other CRAN checks

**Pull request merged**